### ISSUE TITLE
Handling of monospaced table cells

### ISSUE THREAD

**Issue by sdavids** [asciidoctor/asciidoctor-intellij-plugin#1601]
## Observed vs. expected behavior

The baseline is incorrect and the color does not adjust to the OS' dark mode.

### JCEF Browser

![light](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/201931/866c02e7-343f-4dd7-8da2-c929c940b4ca)

![dark](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/201931/d9069fd7-46b1-4442-aed9-7168794df79e)

![baseline](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/201931/03db6e82-8d6b-487a-9737-213ea7e3a27b)

## Reference

[AsciiDoc - Format Content by Cell](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/)

## Steps to reproduce

```asciidoc
[%header,cols=",^m,<m,>m"]
|===

|A | B | C | D

|1
|2
|3
|4

|===
```

## Environment

Plugin Version: 0.41.13

IntelliJ IDEA 2024.1 (Community Edition)
Build #IC-241.14494.240, built on March 28, 2024
Runtime version: 17.0.10+8-b1207.12 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.7.4
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
Non-Bundled Plugins:
  org.asciidoctor.intellij.asciidoc (0.41.13)
Kotlin: 241.14494.240-IJ

**Comment by ahus1** [comment]
@sdavids - thank you for raising this issue. The dark mode you see is part of the AsciiDoc plugin, and I'll take care of it in this issue. 

The vertical alignment issue you reported here is part of the CSS stylesheet maintained by the Asciidoctor project. Please report the issue against that project: https://github.com/asciidoctor/asciidoctor

**Comment by ahus1** [comment]
Thank you for sharing the reproducer. The next version of the plugin will render this better. 

The following example:

```
test is `a` test

[%header,cols=",^m,<m,>m"]
|===

|A | B | C | D

|1
|2
|3
|4

|===
```

Before:

![image](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/3957921/b1f97af2-d2f8-41b4-872a-741f59d0bf7a)

After: 

![image](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/3957921/c613848c-177a-489b-ac1e-5b553bfc522d)

I'll update this issue once a preview version is available.

**Comment by ahus1** [comment]
The changes are available as part of version 0.41.15 of the plugin which is currently available as a pre-release.

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
